.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,72 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ "main", develop, staging ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ "main" ]
+  schedule:
+    - cron: '25 21 * * 0'
+
+jobs:
+  analyze:
+    name: Analyze
+    runs-on: ubuntu-latest
+    permissions:
+      actions: read
+      contents: read
+      security-events: write
+
+    strategy:
+      fail-fast: false
+      matrix:
+        language: [ 'javascript' ]
```

Comment:
it [supported](https://aka.ms/codeql-docs/language-support) `typescript`

Reply:
typescript is trigger by default, you can see the log

Reply:
Tin chuẩn chưa anh
![239997824_1673803459488529_3231250185561427304_n](https://user-images.githubusercontent.com/23650671/177247913-29bb3030-ee35-47ea-843b-87b6d0a08b6b.jpeg)

Reply:
definitely "Chuẩn"
![Uploading image.png…]()